00-HTML-CSS-basics/css/2-2/css/styles.css

```diff
@@ -0,0 +1,15 @@
+html 
+{ 
+	font-family: Arial, Helvetica, sans-serif;
+	font-size: 14px;
+}
+
+header, footer, aside, nav 
```

Comment:
As a general code styling comment, grouped selectors should be separated by a line break and curly braces should always be right next to the definition, separated by one space:
```
header,
footer,
aside,
nav {
    background-color: #435098;
}
```